tao_compiler/mlir/disc/disc.bzl

```diff
@@ -62,3 +62,9 @@ def if_disc_x86(if_true, if_false=[]):
         "//tensorflow/compiler/mlir/disc:disc_x86": if_true,
         "//conditions:default": if_false
     })
+
+def if_torch_disc(if_true, if_false=[]):
+    return select({
+        "//tensorflow/compiler/mlir/disc:is_torch_disc": if_true,
```

Comment:
As I guess, `is_torch_disc` should be set to `true` when building for torch and `false` when for tf?
I thinks this situation could be improved to support both torch/tf in a single bazel configuration.

Reply:
Yes, set `is_torch_disc` makes toch-blade‘s entire build actions no need to directly call `cmake`. Since `tf` side's compiler still build in `org_tensorflow` workspace, so we need `is_torch_disc ` to set them apart.

Reply:
But this behavior makes our build processor more complicated. Considering a developer just configure bazel build once for the whole repo with device/GCC/.. and other options and then `bazel build //tao` / `bazel build //torch_blade` / `bazel build //tf_blade` in any order. Setting an option before building torch_blade will introduce extra cognitive load to developer, while it is avoidable...

Reply:
This is no longer needed anymore

Reply:
But still used at `tao_compiler/mlir/xla/ral/BUILD:250` in this PR.